src/main/java/io/vertx/ext/apex/RoutingContext.java

```diff
@@ -322,4 +323,19 @@
    */
   void setAcceptableContentType(String contentType);
 
+  /**
+   * @return Get the current locale {@link java.util.Locale}. The context must have first been routed to a
+   * {@link io.vertx.ext.apex.handler.LocaleHandler} for this to be populated.
+   */
+  Locale getLocale();
```

Comment:
Stephane,

Did you build the project and run the tests after making your changes? (At command line).

I doubt if it will compile as Locale is not a class supported by codegen...